marketplaceItems/sitPoint/appResources/appData/sitServer.js

```diff
@@ -14,38 +14,36 @@
 /* globals Entities Script */
 
 (function() {
-
-    var isOccupied;
-    var entityID = null;
-    var currentClientSessionID = null;
-
     var HEARTBEAT_INTERVAL_TIME_MS = 10000; // ms
     var RESOLVED_TIMEOUT_TIME_MS = 1000; // ms
     
-    var resolved = false;
-    var heartbeatInterval = null;
-
+    var that = null;
     function SitServer() {
-
+        that = this;
+        this.isOccupied;
```

Comment:
you want these declarations to be inside your prototype instantiation's preload, not here. (see Slack)

Reply:
This is how you assign variables to each instance of SitServer.